examples/5_secret_storage.py

```diff
@@ -18,6 +18,8 @@
 #
 # Here we've stored the HMAC secret and other entries in the team's kvstore;
 # you could also store the entries in the bot's own kvstore (the default team).
+#
+# TODO: this is currently not working exactly right. sorry.
```

Comment:
should we file a ticket w/ more details?

Reply:
sure. i'll do it.